commit de4611bd88245d5ae4eadc419c2779136e39f6dc
Author: notro <notro@tronnes.org>
Date:   Sun Jul 27 20:14:05 2014 +0200

    spi: bcm2835: make driver available on ARCH_BCM2708
    
    Make this driver available on ARCH_BCM2708
    
    Signed-off-by: Noralf Tronnes <notro@tronnes.org>

diff --git a/drivers/spi/Kconfig b/drivers/spi/Kconfig
index cb20594..54aac0f 100644
--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -76,7 +76,7 @@ config SPI_ATMEL
 
 config SPI_BCM2835
 	tristate "BCM2835 SPI controller"
-	depends on ARCH_BCM2835 || COMPILE_TEST
+	depends on ARCH_BCM2835 || ARCH_BCM2708 || COMPILE_TEST
 	help
 	  This selects a driver for the Broadcom BCM2835 SPI master.
 
